server/server_test.go

```diff
@@ -390,8 +392,32 @@ func TestRuleNamesResponse(t *testing.T) {
 			Endpoint:           server.RuleIDs,
 			AuthorizationToken: goodJWTAuthBearer,
 		}, &helpers.APIResponse{
-			StatusCode: http.StatusOK,
-			Body:       expectedBody,
+			StatusCode:  http.StatusOK,
+			Body:        expectedBody,
+			BodyChecker: ruleIDsChecker,
 		})
 	}, testTimeout)
 }
+
+func ruleIDsChecker(t testing.TB, expected, got []byte) {
+	type Response struct {
+		Status string   `json:"status"`
+		Rules  []string `json:"rules"`
+	}
+
+	var expectedResp, gotResp Response
+
+	if err := json.Unmarshal(expected, &expectedResp); err != nil {
+		err = fmt.Errorf(`"expected" is not JSON. value = "%v", err = "%v"`, expected, err)
+		helpers.FailOnError(t, err)
+	}
+
+	if err := json.Unmarshal(got, &gotResp); err != nil {
+		err = fmt.Errorf(`"got" is not JSON. value = "%v", err = "%v"`, got, err)
+		helpers.FailOnError(t, err)
+	}
+
+	sort.Strings(expectedResp.Rules)
+	sort.Strings(gotResp.Rules)
+	assert.Equal(t, expectedResp, gotResp)
```

Comment:
btw, there some function in assert package which does the same as these 3 lines

Reply:
Mmm, interesting. I discovered a lot of new asserting functions on `assert` package, but I cannot find the one you are referring to.

The most likely for me is using 2 `assert.Subset` (`expectedResp` is a subset of `gotResp` and vice versa).

Reply:
`assert.ElementsMatch` is the answer. Nothing like writting a comment asking for something to find it by yourself

Reply:
Updated commit using `assert.ElementsMatch` instead of sorting by myself